src/main/target/STM32_UNIFIED/target.h

```diff
@@ -78,6 +78,33 @@
 #define TARGET_IO_PORTD 0xffff
 #define TARGET_IO_PORTE 0xffff
 
+#elif defined(STM32F411_BMI270)
```

Comment:
Thes should rather be built as sub-clauses of `STM32F411`:
```
#elif defined(STM32F411) || defined(STM32F411_BMI270)
#if defined(STM32F411)
#define TARGET_BOARD_IDENTIFIER "S411"

#define USBD_PRODUCT_STRING     "Betaflight STM32F411"
#else
#define TARGET_BOARD_IDENTIFIER "S41B"

#define USBD_PRODUCT_STRING     "Betaflight STM32F411_BMI270"
#endif

#define USE_I2C_DEVICE_1
[...]
```

Reply:
ok, will change. Much appreciate